libdnf.spec

```diff
@@ -31,6 +31,7 @@ Source0:        %{url}/archive/%{version}/%{name}-%{version}.tar.gz
 
 BuildRequires:  cmake
 BuildRequires:  gcc
+BuildRequires:  gcc-c++
```

Comment:
Please change ``%global dnf_conflict 2.8.7`` to ``%global dnf_conflict 3.0.0``

Reply:
fixed